ns1/resource_datafeed.go

```diff
@@ -35,25 +39,39 @@ func dataFeedResource() *schema.Resource {
 }
 
 func dataFeedToResourceData(d *schema.ResourceData, f *data.Feed) {
+	configAdapterOut(f)
 	d.SetId(f.ID)
 	d.Set("name", f.Name)
 	d.Set("config", f.Config)
 }
 
-func resourceDataToDataFeed(d *schema.ResourceData) *data.Feed {
+func resourceDataToDataFeed(d *schema.ResourceData) (feed *data.Feed, e error) {
+	err := configAdapterIn(d)
+	if err != nil {
+		return nil, err
+	}
 	return &data.Feed{
 		Name:     d.Get("name").(string),
 		SourceID: d.Get("source_id").(string),
 		Config:   d.Get("config").(map[string]interface{}),
-	}
+	}, nil
 }
 
 // DataFeedCreate creates an ns1 datafeed
 func DataFeedCreate(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*ns1.Client)
+<<<<<<< HEAD
```

Comment:
Looks like there are some artifacts left from a git merge in here that should be removed.

Reply:
Ops, my bad, I Forgot to save the merge before send